src/model_output_summary.R

```diff
@@ -0,0 +1,60 @@
+model_output_summary <- function(target_date){
+  ## A function to write a summary table of `model-output` and `model-metadata`
+  ## to PDF format. 
+  # Input: target_date = a model submission date in date format. 
+  
+  library(arrow)
+  library(yaml)
+  library(knitr)
+  library(kableExtra)
```

Comment:
I'm getting a message on this branch
```
> renv::status()
The following package(s) are used in this project, but are not installed:
- kableExtra
```
So I think that the kableExtra package needs to be added to the renv lockfile.

Reply:
In the current version there are two library calls outside of the function. I think we have two options here:
1. use `require()` inside of the function to load the libraries, e.g. `require(arrow)`.
2. use the syntax where it is specifically needed, like `arrow::read_parquet()`.
But having the library calls outside of the function should be changed.

Reply:
> use the syntax where it is specifically needed, like `arrow::read_parquet()`

Use this syntax. The problem with `library()` and `require()` is that they attach all exported functions of these packages.

Reply:
I modified the file so that it uses the package::function() syntax and removed the library calls!